src/style.css

```diff
@@ -1,3 +1,93 @@
-body {
+html body {
+  font-family: Roboto, Arial, sans-serif;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  --header-active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  flex-shrink: 0;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header .logo img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+  margin-left: 50px;
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 50px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline-start: 0;
+}
+
+.nav_item {
```

Comment:
```suggestion
.nav__item {
```

Reply:
Thank you Darek for code review. I have just posted of push called "
added task solution Moyo-header v2 after code review by darokrk". 
Take a look and give further suggestions. If possible, please.
Andrzej